templates/nginx/galaxy-main.j2

```diff
@@ -140,7 +140,7 @@ server {
 		proxy_set_header         Upgrade $http_upgrade;
 		proxy_set_header         Connection "upgrade";
 		client_max_body_size     0;
-		proxy_pass http://upload.galaxyproject.eu:1081/files;
+		proxy_pass http://upload.galaxyproject.eu:1081/api/upload/resumable_upload;
```

Comment:
When you create an upload, rustus tells you that it is available in the URL that you set in `proxy_pass`, rather than the URL that you provided.